src/Symfony/Component/Console/Tests/ApplicationTest.php

```diff
@@ -414,8 +414,10 @@ public function testSetCatchExceptions()
         }
     }
 
-    public function testAsText()
+    public function testLegacyAsText()
     {
+        $this->iniSet(-1 & E_USER_DEPRECATED);
```

Comment:
aren't you missing an argument here sayign it should set the error reporting at this value ?